eng/tools/sdk-suppressions/src/index.ts

```diff
@@ -0,0 +1,31 @@
+
+import { exit } from "process";
+import { updateSdkSuppressionsLabels, PullRequestContext } from "./updateSdkSuppressionsLabel.js";
+
+function getUsage(): string {
+    return (
+      "Parameter missing! \n" + 
+      "Check GITHUB_PULL_REQUEST_CONTEXT where defined in step: Get GitHub PullRequest Context" +
+      "Check OUTPUT_FILE in step env: Run get suppressions label script " +
+      "Check The pr has change files in GITHUB_PULL_REQUEST_CHANGE_FILES "
```

Comment:
You will need to determine the changed files at runtime from the base and head commit params.  Example:

https://github.com/Azure/azure-rest-api-specs/blob/1e620cfbf7df188acef4d6b4a8752aa3aa82fa02/eng/scripts/TypeSpec-Requirement.ps1#L55-L57

Reply:
changed